rho/facts.py

```diff
@@ -100,149 +100,149 @@ def new_fact(fact_name, description, is_sensitive=False,
             category.add(fact_name)
 
 
-new_fact('connection.host', 'The host address of the connection',
+new_fact('connection.host', 'The host address of the connection', True,
```

Comment:
You don't have to change it, but this may be a case where using the parameter name explicitly for everything might improve the readability of choices as we add more facts.
Example:
```
new_fact(fact_name='connection.host', description='The host address of the connection', is_default=True, is_sensitive=True, always_collect=True)
```

Reply:
I was thinking about whether to do that. Since you also mention, I will make the change.

I'd rather leave the fact name and description as they are, because I think they are more clear from context. But definitely `is_default=True` is a lot more readable than some random boolean in an argument list.